1-Draft/RFCnnnn-Alias-Policy.md

```diff
@@ -0,0 +1,108 @@
+---
+RFC: nnnn
+Author: Ilya Sazonov
+Status: Draft
+Area:
+Comments Due: 6/30/2018
+---
+
+# The document purpose
+
+The purpose of this document is to establish a new policy for the creation and use of aliases to ensure:
+
+* Reliable, predictable behavior of scripts and modules
+* Increase readability and maintainability of scripts and modules
+* UX enhancements in interactive sessions
+
+# Problem description
+
+1. Windows PowerShell has many aliases that conflict with the names of standard Unix utilities.
+
+    These aliases was removed from PowerShell Core.
+    This causes a backward compatibility issue when Windows users do not find the familiar aliases in the interactive session.
+    It also disrupts the operation of custom scripts that use these aliases.
+
+1. Now there are dozens of modules developed by MSFT and thousands of modules of third developers. This causes a name conflict.
+
+    Developers try to create unique cmdlet names.
+    They can use module prefixes to resolve name conflicts.
+
+    On the other hand, developers of the best intentions can complement their modules cmdlet aliases.
+    At the same time, users can create their own aliases.
+
+    With a large number of existing modules and cmdlets and a wide variety of custom aliases, these short names inevitably conflict with each other.
+
+    Moreover, the behavior becomes unpredictable because each subsequent download module or script can change any alias.
+    Either the user can destroy a script behavior by creating a new alias or by deleting an existing one.
+
+    It is even more unpredictable and dangerous to create scripts in the assumption that there are some predefined (standard) aliases because they can be changed by the user or script at any time.
+
+    Modern versions of ScriptAnalyzer already recommend replacing aliases in scripts with their full names.
+
+    >The definition of aliases in modules or scripts and their automatic import becomes an undesirable and dangerous practice.
+
+1. UX in interactive sessions.
+
+    Having many cmdlets and parameters with long names can make it difficult to enter them in interactive sessions.
+    Users who typically use standard settings are typically satisfied with the ```IntellySense``` capabilities.
+    On the other hand, some users actively using interactive sessions may still be annoyed by the need to type names.
```

Comment:
I believe PowerShell is **primarily** about _interactive sessions_.  The phrasing in this section makes it sound as though you think interactive sessions are a secondary mode that only a few people use...